### DISCUSSION TITLE
Reorganize dashboard's metrics

### DISCUSSION THREAD

**Discussion by DarkrysVakarian** [swizzin/swizzin#848] [Q&A + Help] [answered]
Hello,

I have a question: is it possible to reorganize the metrics on the dashboard?

I have the impression that at the installation they were put in the "wrong" order : 
<img width="1430" alt="scrennshot" src="https://user-images.githubusercontent.com/22872351/157960490-74369ac5-c294-4c9a-9603-fb863088464b.png">

Thnak you :)

**Comment by liaralabs** [comment, marked as answer]
No, they are static.

It likely looks the way it does because your viewport isn't tall enough to accommodate the lengthy list of services you have installed underneath the "At a Glance" section which pushes it over into the next column.

**Reply by DarkrysVakarian** [reply]
Oh okay I see.

Thanks for your answer :)
